src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -172,12 +173,23 @@ const AuctionResultsContainer: React.FC<AuctionResultsProps> = ({
         </AuctionFilterMobileActionSheet>
       )}
       <Media greaterThan="xs">
-        <Flex justifyContent="space-between" alignItems="flex-start" pb={4}>
-          <Text variant="xs" textTransform="uppercase">
-            Filter by
-          </Text>
-          <SortSelect />
-        </Flex>
+        <Row>
+          <Col sm={3} pr={[0, 2]}>
```

Comment:
These column components are deprecated in favor of [GridColumns](https://github.com/artsy/palette/blob/master/packages/palette/src/elements/GridColumns/GridColumns.story.tsx).

(Are you using VSCode? If so, it should cross out these components as deprecated,)